wire-ios-request-strategy/Sources/Helpers/ZMSound.swift

```diff
@@ -76,7 +76,9 @@ public enum ZMSound: String, CustomStringConvertible {
         switch self {
         case .None:
             return nil
-        case .WireCall, .WirePing, .WireText:
+        case .WireCall,
```

Comment:
I'll go with like or not enforces, worst case a line length would enforce going to the next line